src/Redist/Microsoft.ML.TensorFlow.Redist/Microsoft.ML.Tensorflow.Redist.proj

```diff
@@ -0,0 +1,114 @@
+<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
```

Comment:
>   [](start = 0, length = 2)

question : do we have a need to package the gpu-libraries as well ?

Reply:
Not in this PR at least.  TF only appears to publish the linux GPU build and the files have the same name as CPU build.  We can consider doing something in the future, but I'd hope we have a better story than linux-only.  Perhaps when we build them ourselves.  Then we need to decide the user gesuture for selection: is it a property set in the project?  package-reference?

---
In reply to: [212439593](https://github.com/dotnet/machinelearning/pull/720#discussion_r212439593) [](ancestors = 212439593)